SkillsTraining/Assets/Scripts/OpenAIApi.cs

```diff
@@ -0,0 +1,34 @@
+using Azure;
+using Azure.AI.OpenAI;
+using static System.Environment;
+
+
+string endpoint = "https://usecase5.openai.azure.com/";
+string key = "f89e9e38f93448d8a99312bd95f5fd7e";
```

Comment:
Not sure it is a good idea to put a key into a public code repo...is there a more secure way to handle this? Azure Secrets call?

Reply:
If I understand this (https://learn.microsoft.com/en-us/azure/search/search-security-api-keys?tabs=rest-use%2Cportal-find%2Cportal-query) correctly, the best practice seems to be switching to a Role-based system - but I have no idea if this is something we can control, I've never used Azure before.